ci/run-locust.yaml

```diff
@@ -0,0 +1,56 @@
+platform: linux
```

Comment:
This file should be called `run-benchmark.yaml`.

Reply:
It originally was, but the implementation is locust specific.

Reply:
Only `locustOptions` is locust specific, and that should also be fixed in another pr.